script/gc-amdvlk-docker-images.py

```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env python3
+"""Script to garbage collect old amdvlk docker images created by the public CI on GitHub.
+
+Requires python 3.9 or later.
+"""
+
+import argparse
+import json
+import logging
+import subprocess
+import sys
+
+from collections import defaultdict
+from typing import Any, Optional
+
+def _run_cmd(cmd: list[str]) -> tuple[bool, str]:
```

Comment:
Would it make sense for `_run_cmd` to return an `Optional[str]` instead?

Reply:
Previously I had it return (ok, stdout, stderr), but decided we don't need to return stderr for now. If we ever need to process errors in the future, the current format is more amenable to that, so I'm leaning towards keeping it as-is.